### ISSUE TITLE
Chromium 81 and next releases

### ISSUE THREAD

**Issue by jbvsmo** [alixaxel/chrome-aws-lambda#124]
## What would you like to have implemented?
Chrome 81 has been released a month ago but not available here.
Is there a guarantee this project will be kept up to date? Do you have an automated script for updating chrome versions?

Also puppeteer is on version 3.0.2. Is there any impediment for upgrading?

## Why would it be useful?
Having latest browser with all features and security

**Comment by ryanblock** [comment]
Issue appears to be duplicative of https://github.com/alixaxel/chrome-aws-lambda/issues/120

**Comment by alixaxel** [comment]
@jbvsmo 

> Is there a guarantee this project will be kept up to date?

As a MIT-licensed OSS, there's no guarantees. :slightly_smiling_face: I try to keep this project up-to-date, and usually it's never more than 1 or 2 days behind, but with the puppeteer team changing and the increasing number of issues being reported on this project, it's getting increasingly hard to follow up in a timely manner.

> Do you have an automated script for updating chrome versions?

Yes, check the [**Compiling** section of the README](https://github.com/alixaxel/chrome-aws-lambda#compiling).

Closing as duplicate of #120.

**Comment by alixaxel** [comment]
I will be releasing a new version shortly BTW.

**Comment by steida** [comment]
@alixaxel Did you consider using Playwright?

**Comment by alixaxel** [comment]
@steida 

Ermm.... Yes, but I haven't figured out how to support both and not break backwards compatibility. :sweat_smile:
